core/runtime/CMakeLists.txt

```diff
@@ -70,6 +70,31 @@ add_library(grandpa_api
     impl/grandpa_impl.cpp
     )
 target_link_libraries(grandpa_api
+    buffer
+    logger
+    scale
+    scale
+    wasm_executor
```

Comment:
Please check if only linking wasm executor is enough. And why there is two scales?

Reply:
artifact of merging:(

Reply:
wasm executor + scale are enough in most cases, only wasm executor is not enough